### PR TITLE
Click or double-click on a selected tile triggers zoom. Resolves GH-1925

### PR DESCRIPTION
**Pull request by thisandagain**

### CODE REVIEW

File: www_src/pages/project/project.jsx [hunk 1]
```diff
@@ -210,11 +210,15 @@ var Project = React.createClass({
...
+  zoomToPage: function (coords, play) {
+    if (typeof play === 'undefined') {
+      play = true;
+    }
+
     this.setState({
       camera: this.cartesian.getFocusTransform(coords, 1),
       zoom: 1,
-      isPageZoomed: true,
+      isPageZoomed: play,
```
Comment (gvn) [inline www_src/pages/project/project.jsx]: Shouldn't this still be `true`?

Comment (gvn) [inline www_src/pages/project/project.jsx]: Nevermind...derp

Comment (gvn) [inline www_src/pages/project/project.jsx]: This is kind of weird...should always be `true` given the method's purpose. There's already `zoomFromPage` to zoom out.

Comment (thisandagain) [inline www_src/pages/project/project.jsx]: I'm not using this to zoom out, this triggers the "zoom out" control at the bottom of the screen and overwrites the edit controls.

**Comment by gvn** [comment]
R+! :ghost:

**Pull request merged**